src/main.js

```diff
@@ -535,7 +649,12 @@ export async function processActions(actions, patientReference, resolver, aux, e
       if (activityDefinition?.relatedArtifact) {
         for (const relatedArtifact of activityDefinition.relatedArtifact) {
           if (relatedArtifact.url !== undefined) {
-            payloads.push({ contentAttachment: { url: relatedArtifact.url, title: relatedArtifact.display }});
+            payloads.push({
```

Comment:
The number of styling changes throughout this PR makes it difficult to understand what exactly is changing. Can we consider removing all of the styling changes from this PR and moving them to a separate PR dedicated only towards styling, if it's something we feel strongly about fixing?

Reply:
Formatting changes have been removed in https://github.com/ccsm-cds-tools/encender/pull/15/commits/891bfbdc776bd725b439903ead71b6f733ff423e.